### ISSUE TITLE
"Please wait while the accessibility scan is running ..." never completes

### ISSUE THREAD

**Issue by mikeriley131** [storybookjs/storybook#9471]
**Describe the bug**

I've set everything up as shown in the README, and I see the Accessibility tab in Storybook now, however it's been showing "Please wait while the accessibility scan is running ..." for almost two hours now for all stories. Is this expected or did I miss something?

**Expected behavior**
I would expect the scan to take no more than 15 minutes or so on initial install.

**System:**
```
System:
    OS: macOS Mojave 10.14.6
    CPU: (12) x64 Intel(R) Core(TM) i9-8950HK CPU @ 2.90GHz
  Binaries:
    Node: 10.17.0 - ~/.nvm/versions/node/v10.17.0/bin/node
    npm: 6.11.3 - ~/.nvm/versions/node/v10.17.0/bin/npm
  Browsers:
    Chrome: 79.0.3945.117
    Firefox: 72.0.1
    Safari: 13.0.4
  npmPackages:
    @storybook/addon-a11y: 5.3.3 => 5.3.3
    @storybook/addon-actions: 5.2.8 => 5.2.8
    @storybook/addon-console: 1.2.1 => 1.2.1
    @storybook/addon-knobs: 5.2.5 => 5.2.5
    @storybook/addon-links: 5.2.8 => 5.2.8
    @storybook/addon-notes: 5.2.8 => 5.2.8
    @storybook/addons: 5.2.8 => 5.2.8
    @storybook/react: 5.2.8 => 5.2.8
```

**Comment by shilman** [comment]
Did you set the `withA11y` decorator`?

**Comment by mikeriley131** [comment]
:)
I had added it to our config.js file but after further inspection it turns out we somehow have two .storybook folders and I had added it to the wrong one. All sorted. Thanks.

**Comment by shilman** [comment]
@mikeriley131 You're not the first person to run into this. I'm re-opening it in case one of the `addon-a11y` maintainers can provide a more useful message.

**Comment by cloudsociety** [comment]
I missed adding the decorator too. Looks like having a `preset.js` file inside addon-a11y would do the trick (similar to addon-knobs). Then, it could be configured directly inside `main.js`.

**Comment by shilman** [comment]
@cloudsociety that would be  great! Any interest in adding one?

**Comment by CodeByAlex** [comment]
@cloudsociety i think that would be very helpful

**Comment by cloudsociety** [comment]
@CodeByAlex 

I can give it a try. Should I be branching from `next`? Seems like it's the default, and being used for 6beta work.

**Comment by CodeByAlex** [comment]
Yeah you’ll want to fork from next

**Comment by shilman** [comment]
Ooh-la-la!! I just released https://github.com/storybookjs/storybook/releases/tag/v6.0.0-alpha.5 containing PR #9697 that references this issue. Upgrade today to try it out!

You can find this prerelease on the `@next` NPM tag.

Closing this issue. Please re-open if you think there's still more to do.

**Comment by chemoish** [comment]
> However, if you are upgrading

https://github.com/storybookjs/storybook/pull/9697/files#diff-177dcef0a5b481c2c87db87f7cdf200dR106 (Sentence that stopped halfway)

Even though the documentation states 5.3+, it is not accurate. Looks like 5.3 still requires decoration configuration.  Would be great to specify requirements for that version.

```tsx
import {addDecorator} from '@storybook/react';
import {withA11y} from '@storybook/addon-a11y';

addDecorator(withA11y);
```

**preview.js**

**Comment by gaurav5430** [comment]
Yeah, i am on 5.3.19 and had to add the decorator in `preview.js`

**Comment by justiceajah** [comment]
> Hello
> 
> Is this problem happening again after [this](https://github.com/storybookjs/storybook/pull/30459)? I come from Storybook's 8.5.3 version and I've just reactivated the add on in the `.storybook/main.ts` file and now it works without the 'vite' annoying error solved by the 8.5.4 patch but there's no accessibility to be seen on the Accessibility tab. Only this message forever:
> 
> ```
> Please wait while the accessibility scan is running ...
> ```
> 
> You can kind-of-fix-it by adapt the configuration to 'manual'. Which is completely undesired.
> 
> ```
> const preview: Preview = {
>   parameters: {
>     a11y: {
>       manual: true
>     }
>   }
> };
> ```
> 
> Maybe there's something more to be done to make it run automatically like before?
> 
> Thanks.

I confirm this issue in version 8.5.5
